src/Search/FlagrowIOSearcher.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+namespace Flagrow\Bazaar\Search;
+
+use Flagrow\Bazaar\Extensions\Extension;
+use Flarum\Core\Search\SearchResults;
+use Flarum\Extension\ExtensionManager;
+use Flarum\Settings\SettingsRepositoryInterface;
+use GuzzleHttp\Client;
+use Illuminate\Database\Eloquent\Collection;
+
+class FlagrowIOSearcher extends AbstractExtensionSearcher
```

Comment:
Let's create some generic packages for our future projects first. That should solve this issue by implementing a Json Api Spec valid paginator.